python_modules/libraries/dagster-ssh/dagster_ssh/resources.py

```diff
@@ -150,7 +181,7 @@ def get_tunnel(self, remote_port, remote_host="localhost", local_port=None):
             local_bind_address = ("localhost",)
 
         # Will prefer key string if specified, otherwise use the key file
-        pkey = self.key_obj if self.key_obj else self.key_file
+        pkey = self._key_obj if self._key_obj else self.key_file
```

Comment:
I think we should add some kind of logging if `_key_obj` and `key_file` are both provided so that the user knows we are deferring to `_key_obj`?